calebwagner/button-the-clown
# added clown selection drop down

I added a drop down selection element to pick a clown and then mark the request as complete in the database.

Steps to check code:
1. git fetch --all
2. git checkout cjw-select-clown
3. code .
4. check Requests.js module for event listener 
5. serve
6. check browser for selection tab of service request
7. select a clown
